flow/benchmarks/rllib/ars_runner.py

```diff
@@ -1,9 +1,10 @@
-"""
-Runner script for environments located in flow/benchmarks.
+"""Runs the environments located in flow/benchmarks.
 
 The environment file can be modified in the imports to change the environment
-this runner script is executed on. Furthermore, the rllib specific algorithm/
-parameters can be specified here once and used on multiple environments.
+this runner script is executed on. This file runs the ARS algorithm in rllib
+and utilizes the hyper-parameters specified in:
+
+TODO: add paper reference
```

Comment:
Simple random search provides a competitive approach to reinforcement learning
by Mania et. al